app/Activity.php

```diff
@@ -6,5 +6,15 @@
 
 class Activity extends Model
 {
-    //
+    protected $fillable = ['user_id', 'activity_id', 'activity_type'];
+
+    public function activity_morph()
```

Comment:
Please use camelCase() for methods as mention on the PSR-1 (standards for php code)
https://www.php-fig.org/psr/psr-1/#43-methods

Reply:
I modified function name `activity_morph` to `activityMorph` on these file.
- `app\Activity.php`
- `resources\views\components\user-activity.blade.php`